src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
don't create UserService userService each time when method login is called
make it class-level variable

Reply:
How I can call method from UserService, when i don`t have member of this class?

Reply:
Can I make this method static?